docs/install/kubernetes.md

```diff
@@ -107,180 +125,129 @@ are being pulled locally for the first time (it will take an average of 10-15 mi
 your internet speed).
 ```
 
-To install the chart with the release name `mlrun-kit` use the following command.
+To install the chart with the release name `mlrun-ce` use the following command.
 Note the reference to the pre-created `registry-credentials` secret in `global.registry.secretName`:
 
 ```bash
 helm --namespace mlrun \
-    install mlrun-kit \
+    install mlrun-ce \
     --wait \
     --timeout 960s \
     --set global.registry.url=<registry-url> \
     --set global.registry.secretName=registry-credentials \
-    v3io-stable/mlrun-kit
+    --set global.externalHostAddress=<host-machine-address> \
+    mlrun-ce/mlrun-ce
 ```
 
-Where `<registry-url>` is the registry URL which can be authenticated by the `registry-credentials` secret (e.g., `index.docker.io/<your-username>` for Docker Hub).
+Where:
+ - `<registry-url>` is the registry URL which can be authenticated by the `registry-credentials` secret (e.g., `index.docker.io/<your-username>` for Docker Hub).
+ - `<host-machine-address>` is the IP address of the host machine (or `$(minikube ip)` if using minikube).
+
+Once the installation is complete, the helm command prints the URLs and Ports of all the MLRun CE services.
+
+## Installing the Full Version
 
-```{admonition} Installing on Minikube/VM**
-The Open source MLRun kit uses node ports for simplicity. If your Kubernetes cluster is running inside a VM, 
-as is usually the case when using minikube, the Kubernetes services exposed over node ports are not available on 
-your local host interface, but instead, on the virtual machine's interface.
-To accommodate for this, use the `global.externalHostAddress` value on the chart. For example, if you're using 
-the kit inside a minikube cluster (with some non-empty `vm-driver`), pass the VM address in the chart installation 
-command as follows:
+To install the full version of the chart, use the following command:
 
 ```bash
 helm --namespace mlrun \
-    install my-mlrun \
+    install mlrun-ce \
     --wait \
     --timeout 960s \
-    --set global.registry.url=<registry URL e.g. index.docker.io/iguazio > \
+    -f override-full.yaml \
+    --set global.registry.url=<registry-url> \
     --set global.registry.secretName=registry-credentials \
-    --set global.externalHostAddress=$(minikube ip) \
-    v3io-stable/mlrun-kit
+    --set global.externalHostAddress=<host-machine-address> \
+    mlrun-ce/mlrun-ce
 ```
-Where `$(minikube ip)` shell command resolving the external node address of the k8s node VM.
 
-## Installing Kubeflow
+## Configuring Online Feature Store
```

Comment:
## Configuring the online feature store
The MLRun Community Edition now supportsthe  online feature store. To enable it, you need to first deploy a REDIS service that is accessible to your MLRun CE cluster.
To deploy a REDIS service, refer to the following [link](https://redis.io/docs/getting-started/).

When you have a REDIS service deployed, you can configure MLRun CE to use it by adding the following helm value configuration to your helm install command: